andela/travel_tool_front
# #161044635 View travel checklist for 'Open' requests

#### What does this PR do?
This PR displays the travel checklists for open requests

#### Description of Task to be completed?
- Add the `travel checklist` button to `request table menu`
- Display travel checklist on a modal when `travel checklist button` is clicked

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`
- Login and make a `new request`
- Click on the `new request` ellipsis and then click on the `travel checklist button` on the modal
- The travel checklist modal should pop up and show the checklist for the travel request.

#### Any background context you want to provide?
- You need to clone the [backend](https://github.com/andela/travel_tool_back.git) to test this functionality.


#### What are the relevant pivotal tracker stories?
[#161044635](https://www.pivotaltracker.com/story/show/161044635)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
